src/apps/explorer/pages/AppData/EncodePage.tsx

```diff
@@ -207,39 +224,94 @@ const EncodePage: React.FC<EncodeProps> = ({ tabData, setTabData /* handleTabCha
+                  <a href="https://www.npmjs.com/package/json-stringify-deterministic" target="_blank" rel="noreferrer">
+                    deterministic JSON formatter
+                  </a>{' '}
+                  , this way the same content yields always the same <strong>AppData hex</strong>
+                </p>
+                <RowWithCopyButton className="appData-hash" textToCopy={fullAppData} contentsToDisplay={fullAppData} />
+                <p className="disclaimer">Note: Don’t forget to upload this file to IPFS!</p>
+              </>
+            )}
             {ipfsHashInfo && (
               <>
-                <h4>AppData Hash</h4>
+                <h2>🐮 AppData hex</h2>
+                <p>
+                  This is the <code>keccak-256</code> hash of the above document represented in hexadecimal format.
+                </p>
                 <p>
-                  This is the corresponding hash of the above document. Use this in your orders <strong>appData</strong>{' '}
-                  field.
+                  Use this in your{' '}
+                  <a
+                    href="https://github.com/cowprotocol/contracts/blob/main/src/contracts/libraries/GPv2Order.sol#L18"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    CoW Orders appData field
+                  </a>{' '}
+                  .
                 </p>
                 <RowWithCopyButton
                   className="appData-hash"
                   textToCopy={ipfsHashInfo.appDataHex}
                   contentsToDisplay={ipfsHashInfo.appDataHex}
                 />
-                <p className="disclaimer">Note: Don’t forget to upload this file to IPFS!</p>
+                <h2>🌍 IPFS CiD</h2>
+                <p>
+                  This is the{' '}
+                  <a href="https://docs.ipfs.tech/concepts/content-addressing/" target="_blank" rel="noreferrer">
+                    IPFS CID
+                  </a>{' '}
+                </p>
+                )
+                <p>
+                  This CID is derived from the <strong>AppData hex</strong> (
+                  <a
+                    href="https://github.com/cowprotocol/app-data/blob/main/src/api/appDataHexToCid.ts#L30"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    see here how
+                  </a>
+                  ). You can see how this <strong>AppData hex</strong> is encoded, using the{' '}
+                  <a href={'https://cid.ipfs.tech/#' + ipfsHashInfo.cid} target="_blank" rel="noreferrer">
+                    CID Inspector
+                  </a>{' '}
+                </p>
+                <p>
+                  What this means is that you can derived the IPFS CID from on-chain CoW Orders, and download the JSON
+                  from IPFS network to see the meta-information of that order
```

Comment:
Also, the backend now hosts and serves this info.
Should that be mentioned here?